mpsl/CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Changes
 =======
 
 * The MPSL radio notification API has been removed. (DRGN-20447)
+* Added the :kconfig:option:`CONFIG_MPSL_LOW_PRIO_IRQN` Kconfig option to configure the low_prio_irq. (DRGN-21019)
```

Comment:
```suggestion
* Added the :kconfig:option:`CONFIG_MPSL_LOW_PRIO_IRQN` Kconfig option to configure the ``low_prio_irq`` interrupt argument. (DRGN-21019)
```
Or something along these lines to make it clear what the change is about.

Reply:
Can you also add `Added the ... promptless Kconfig option`?
So it makes it clear you usually don't want to mess with this.

Reply:
TBH, I'm not sure if adding "promptless Kconfig option" makes it more clear. Then I would rather add a new sentence below explaining how the user should be able to change it

Reply:
thing is, it's nice to keep it promptless so the transition to devicetree can be seamless.
If users start messing with this now, they then will have even more work to do when we move to dts later.

From my understanding, it's only one (internal to sdk-nrf) user that asked for it, yeah? Then honestly, I wouldn't even mention it in the release notes.